spec/models/manageiq/providers/embedded_ansible/automation_manager/job_spec.rb

```diff
@@ -3,14 +3,199 @@
 
   before do
     region = MiqRegion.seed
-    allow(region).to receive(:role_active?).with("embedded_ansible").and_return(role_enabled)
     allow(MiqRegion).to receive(:my_region).and_return(region)
   end
 
   context 'when embedded_ansible role is enabled' do
-    let(:role_enabled) { true }
+    before do
+      EvmSpecHelper.assign_embedded_ansible_role
+    end
+
+    let(:ansible_script_source) { FactoryBot.create(:embedded_ansible_configuration_script_source, :manager => manager) }
+    let(:playbook)              { FactoryBot.create(:embedded_playbook, :configuration_script_source => ansible_script_source) }
+    let(:manager)               { FactoryBot.create(:embedded_automation_manager_ansible, :provider) }
+
+    let(:machine_credential)    { FactoryBot.create(:ansible_machine_credential, :manager_ref => '1', :resource => manager) }
+    let(:cloud_credential)      { FactoryBot.create(:ansible_cloud_credential,   :manager_ref => '2', :resource => manager) }
+    let(:network_credential)    { FactoryBot.create(:ansible_network_credential, :manager_ref => '3', :resource => manager) }
+    let(:vault_credential)      { FactoryBot.create(:ansible_vault_credential,   :manager_ref => '4', :resource => manager) }
+
+    let(:raw_stdout_json) do
+      [
+        {"stdout" => "A stdout from the job"},
+        {"stdout" => "Errmahgerd... ANSIBLER"},
+        {"stdout" => "And another one"}
+      ]
+    end
+
+    let(:template) { FactoryBot.create(:embedded_ansible_configuration_script, :manager => manager, :parent => playbook) }
+
+    describe 'job operations' do
+      describe ".create_job" do
```

Comment:
Mixed quote styles ... can we choose just one?

Reply:
Copied over from `manageiq-providers-ansible_tower`, but I can change.  👍